corehq/util/couch_doc_processor.py

```diff
@@ -147,6 +147,42 @@ def should_process(self, doc):
         return True
 
 
+class ProcessorProgressLogger(object):
+    def progress_starting(self, total, previously_visited):
+        print("Processing {} documents{}: ...".format(
```

Comment:
I see you removed the list of doc types being migrated here. That was a pretty useful sanity check output to verify that the selected migration was actually migrating the expected documents. Is that being logged somewhere else? If not, I'd suggest adding an optional `message=""` parameter here (default is empty) so it can be re-added.